.github/ISSUE_TEMPLATE/5_performance_speed.yml

```diff
@@ -0,0 +1,191 @@
+name: My app is slow or missing frames
+description: |
+  You are writing an application but have discovered that it is slow,
+  you are not hitting 60Hz, or you are getting jank (missed frames).
+labels: 'from: performance template'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Flutter!
+
+        If you are looking for support, please check out our documentation
+        or consider asking a question on Stack Overflow:
+
+          - https://flutter.dev/
+          - https://api.flutter.dev/
+          - https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Please tell us exactly how to reproduce the problem you are running into.
+      placeholder: |
+        1. ...
+        2. ...
+        3. ...
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Code sample
+      description: |
+        Please create a minimal reproducible sample that shows the problem
+        and attach it below between the lines with the backticks.
```

Comment:
In the demo page you linked to, it looks like line breaks in the description fields are carried over exactly. In that case we should only be breaking these lines where we actually want new lines (e.g., paragraphs).

Reply:
I think you're referring to the line break between `problem` and `and`, right? If so, I have also updated the descriptions in other sections accordingly (only break a line in the end of the sentence).